olive/common/utils.py

```diff
@@ -253,6 +255,26 @@ def tensor_data_to_device(data, device: str):
         return data
 
 
+def tensor_data_to_dtype(data, dtype):
+    if dtype is None:
```

Comment:
can you move the torch import into this function. We wanted to only keep stdlib imports on the top level for this file.